README.md

```diff
@@ -1,3 +1,58 @@
 # All about web components
 
 This repository brought to you by the [Web Components Community Group](https://www.w3.org/community/webcomponents/); the goal being to collect and share documentation and guides for web component best practices.
+
+# Contributor Guidelines
```

Comment:
Would it make sense to link to the file instead?  Otherwise we will have to remember to duplicate this content anytime we change it in either place, no?

Also by having a _CONTRIBUTING.md_, it should get [prompted to contributors automatically by GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors).